src/comp/comp.py

```diff
@@ -24,48 +24,49 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name.startswith("D")]
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name.endswith("e")]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans if any(human.name.startswith(i) for i in 'CDEFG')]
```

Comment:
You can directly compare letters with <= and => thanks to ASCII tables, so that could be another aproach to check if string starts between two particular letters.